packages/KIcon/icons/index.js

```diff
@@ -5,6 +5,7 @@ import arrowUp from './icn-arrow-up.svg'
 import back from './icn-back.svg'
 import book from './icn-book.svg'
 import brain from './icn-brain.svg'
+import calendar from './icn-calendar.svg'
```

Comment:
This is **super-nit**...

If you compare the new `calendar` icon to the `contactSupport` icon (or any others), the border on the calendar icon looks much thicker with a weird border-radius. Is it possible to get an icon that looks more similar to the others?

![image](https://user-images.githubusercontent.com/2229946/190670412-91b99f50-00a2-4d84-86d3-f1847a397bef.png)

Reply:
I agree, it's a little off.

I think we need a one file that contains artboards (in Illustrator terms) of all the files, so we can have exact view boxes, and be able to tell how line thickness compares from one icon to another. If this file already exists, then you can ignore me :)

CC @jillztom @helen-tang27

Reply:
Punting on this, future pass ;)